src/consensus/default_constants.py

```diff
@@ -59,7 +59,7 @@
     "BLOCKS_CACHE_SIZE": 4608 + (128 * 4),
     "WEIGHT_PROOF_RECENT_BLOCKS": 1000,
     "MAX_BLOCK_COUNT_PER_REQUESTS": 32,  # Allow up to 32 blocks per request
-    "INITIAL_FREEZE_PERIOD": 5000,  # Transaction are disabled first 5000 blocks
+    "INITIAL_FREEZE_PERIOD": 42 * 4608,  # Transaction are disabled first 5000 blocks
```

Comment:
I think we have to have a 5000 or 2500 block to test anything before we have the go/no go decision

Reply:
Yes this is now being overridden in the overrides (config)